migrate_pim/upgrade/upgrade_from_50_to_60.rst

```diff
@@ -37,17 +37,12 @@ Updated crontab definition
 
 Check your crontab is in sync with :doc:`/cloud_edition/flexibility_mode/docs/crontasks`
 
-Enterprise Edition
-^^^^^^^^^^^^^^^^^^
-
-Renamed
-
-- From `pimee:data-quality-insights:schedule-periodic-tasks` to `pim:data-quality-insights:schedule-periodic-tasks`
-- From `pimee:data-quality-insights:evaluate-products` to `pim:data-quality-insights:evaluations`
```

Comment:
There already is a linked list, no need to add a new one with desynchronisation risk.